app-shared/src/androidJvmMain/kotlin/io/github/droidkaigi/confsched/KaigiAppUi.androidJvm.kt

```diff
@@ -41,6 +43,7 @@ import io.github.droidkaigi.confsched.navkey.TimetableNavKey
 context(appGraph: AppGraph)
 actual fun KaigiAppUi() {
     val backStack = rememberNavBackStack(TimetableNavKey)
+    val doForward = rememberOneStepForwardController(backStack)
```

Comment:
https://github.com/DroidKaigi/conference-app-2025/pull/554#discussion_r2325326310
The rememberOneStepForwardController is defined in the expect/actual method and does not affect other platforms such as Android or iOS. There are no issues as expressed in the above comment.
It should not hinder the review process.